layout/Login/Login.tsx

```diff
@@ -16,12 +16,12 @@ import { LoginForm } from "./components";
 function Login() {
   return (
     <div className="min-h-screen select-none overflow-hidden bg-secondary">
-      <Return componentStyle="sm:ml-20 mt-20 sm:mt-20" />
+      <Return componentStyle="sm:ml-20 mt-20 sm:mt-20 z-10 pl-10 sm:pl-0" />
```

Comment:
Hey! Instead of removing the padding on mobile, you removed the padding in all breakpoints except mobile (remember tailwind's breakpoints follow a mobile-first approach). Try this instead:
```suggestion
      <Return componentStyle="sm:ml-20 mt-20 sm:mt-20 z-10 sm:pl-10" />
```